### PR TITLE
FIX: delete thirdparty with llx_societe_account

### PR DESCRIPTION
**Pull request by ptibogxiv**

### CODE REVIEW

File: htdocs/societe/class/societe.class.php [hunk 1]
```diff
@@ -1553,6 +1553,18 @@ public function delete($id, User $fuser = null, $call_trigger = 1)
 				}
 			}
 
+			// Remove third party  TODO trigger for external action linked to these accounts
```
Comment (eldy) [inline htdocs/societe/class/societe.class.php:1556]: Why adding this code ? There exactly the same just after...

Comment (ptibogxiv) [reply htdocs/societe/class/societe.class.php:1556]: @eldy bad paste.. i fixed it... so many phone call...

File: htdocs/societe/class/societe.class.php [hunk 2]
```diff
@@ -1553,6 +1553,18 @@ public function delete($id, User $fuser = null, $call_trigger = 1)
...
+			if (! $error)
+			{
+				$sql = "DELETE FROM ".MAIN_DB_PREFIX."societe_account";
```
Comment (eldy) [inline htdocs/societe/class/societe.class.php:1559]: This is surely a bug. The rowid in this table is not same than rowid in societe.
May be you just wnt to add the table societe_account into the property array  ->childtablesoncascade

Comment (ptibogxiv) [reply htdocs/societe/class/societe.class.php:1559]: @eldy fix done... really need a break :p

**Pull request closed without merging**